kratos/elements/levelset_convection_element_simplex.h

```diff
@@ -111,15 +111,9 @@ class LevelSetConvectionElementSimplex
         if (rRightHandSideVector.size() != TNumNodes)
             rRightHandSideVector.resize(TNumNodes, false); //false says not to preserve existing storage!!
 
-
-//         noalias(rLeftHandSideMatrix) = ZeroMatrix(TNumNodes, TNumNodes);
-//         noalias(rRightHandSideVector) = ZeroVector(TNumNodes);
-
-//         //Crank-Nicholson factor
-//         const double cr_nk = 0.5;
-
         const double delta_t = rCurrentProcessInfo[DELTA_TIME];
         const double dt_inv = 1.0 / delta_t;
+        const double theta = rCurrentProcessInfo.Has(TIME_INTEGRATION_THETA) ? rCurrentProcessInfo[TIME_INTEGRATION_THETA] : 0.5;
```

Comment:
Finally we keep the academic criterion :smile:

Reply:
You win this time!!